copulas/univariate/truncnorm.py

```diff
@@ -38,10 +44,18 @@ def from_dict(cls, parameters):
         """
         instance = cls()
         instance.fitted = parameters['fitted']
-        instance.constant_value = parameters['constant_value']
 
-        if instance.fitted and instance.constant_value is None:
-            instance.model = scipy.stats.truncnorm(parameters['a'], parameters['b'])
+        if instance.fitted:
+            a = parameters['a']
+            b = parameters['b']
+            mean = parameters['mean']
+            std = parameters['std']
+
+            if a == b == mean and std == 0:
```

Comment:
Since we will be removing mean and std, let's encode this as `constant <=> a == b`